### ISSUE TITLE
Focused Launch: modal a11y

### ISSUE THREAD

**Issue by ciampo** [Automattic/wp-calypso#46864]
- [x] Pressing "ESC" key closes the modal
- [x] Focused should be trapped inside the modal

**Comment by alshakero** [comment]
These two seem already in place. Escape closes the modal and the focus cycle is all within the modal. Did you notice otherwise?

**Comment by ciampo** [comment]
My bad. The issue was already fixed by using the Modal from `@wordpress/components`
